doc/classes/PhysicsDirectSpaceState2D.xml

```diff
@@ -25,8 +25,9 @@
 			<param index="0" name="parameters" type="PhysicsShapeQueryParameters2D" />
 			<param index="1" name="max_results" type="int" default="32" />
 			<description>
-				Checks the intersections of a shape, given through a [PhysicsShapeQueryParameters2D] object, against the space. The resulting array contains a list of points where the shape intersects another. Like with [method intersect_shape], the number of returned results can be limited to save processing time.
+				Checks the intersections of a shape, given through a [PhysicsShapeQueryParameters2D] object, against the space. The resulting array contains a list of points where the shape intersects another.
 				Returned points are a list of pairs of contact points. For each pair the first one is in the shape passed in [PhysicsShapeQueryParameters2D] object, second one is in the collided shape from the physics space.
+				The number of returned results can be limited with the [param max_results] parameter, to reduce the processing time. [param max_results] is clamped to a hard limit of [code]2048[/code].
```

Comment:
If we want to minimize the number of similar strings for translation reasons, this string would be usable for all the functions here, since it's somewhat generic.